data/saas/config/adyen_config.yml

```
@@ -0,0 +1,76 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Adyen
+  type: adyen
+  description: A sample schema representing the Adyen integrations for Fides
+  user_guide: https://docs.ethyca.com/user-guides/integrations/saas-integrations/adyen
+  version: 0.1.0
+
+  connector_params:
+    - name: domain_management
+      label: Management domain
+      description: Please note that Adyen has both TEST and LIVE environments. E.g. management-test.adyen.com or management-live.adyen.com, and that API keys for live and test will not be the same.
+    - name: domain_ca
+      label: CA domain
+      description: Please note that Adyen has both TEST and LIVE environments. E.g. ca-test.adyen.com or ca-live.adyen.com, and that API keys for live and test will not be the same.
+    - name: merchant_account
+      label: Merchant account
+      description: Your merchant account.
+    - name: api_key
+      label: API key
+      description: See Adyen's documentation for me details about creating API keys, and how to assign roles for the key here (https://docs.adyen.com/development-resources/api-credentials/).
+      sensitive: True
+
+  external_references:
+    - name: adyen_user_id
+      label: pspReference
```

This should be in sentence case since it's a user-facing label
```suggestion
      label: PSP reference
```